src/main/java/uk/gov/companieshouse/company/links/tranformer/InsolvencyApiTransformer.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.companieshouse.company.links.tranformer;
+
+import org.springframework.stereotype.Component;
+
+
+//TODO check whether we need this transformer or not
```

Comment:
We do need a component to update the links e.g. for each record the company-links-consumer called the company-profile-api to get the company profile. It then determines if a key value should be added to the `links` field. 

@zahidm3 Less complex than delta-consumers - could handle in the main processor class or here. What do you think?

Reply:
@dominicst-ch , Agree, sub-delta consumers are less complex and don't need transformers. @adeshpandech, lets remove it for now

Reply:
thanks, removed now